test/factories/host_related.rb

```diff
@@ -1,13 +1,83 @@
 FactoryGirl.define do
+  factory :ptable do
+    sequence(:name) { |n| "ptable#{n}" }
+    layout 'zerombr yes\nclearpart --all    --initlabel\npart /boot --fstype ext3 --size=<%= 10 * 10 %> --asprimary\npart /     --f   stype ext3 --size=1024 --grow\npart swap  --recommended'
+  end
+
+  factory :parameter do
```

Comment:
Styling issues here, a space in between factories seems like the standard we've been following everywhere else. There are 2 spaces above this line and no spaces between factories from here on. Sorry for the nitpick